recirq/qml_lfe/sample_analysis.ipynb

```diff
@@ -0,0 +1,1451 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "bbdd6c60",
+   "metadata": {
+    "id": "35a4782cd01c"
+   },
+   "source": [
+    "# Learning states with conventional experiment"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "38297cf2",
+   "metadata": {
+    "id": "2eb02bf1e842"
+   },
+   "source": [
+    "Using randomized Pauli measurement from https://www.nature.com/articles/s41567-020-0932-7 to predict which of the two Pauli observables have a higher magnitude of expectation value.\n",
+    "The number of experiments need to scale exponentially with the system size $n$ to make accurate prediction.\n",
+    "\n",
+    "We perform measurement error mitigation as described in the manuscript (Appendix A.2.d)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "c9c8af31",
+   "metadata": {
+    "id": "b68ecbbc6515"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "ACC: 0.465\n"
+     ]
+    }
+   ],
+   "source": [
+    "import numpy as np\n",
+    "from scipy.stats import moment\n",
+    "import matplotlib.pyplot as plt\n",
+    "import seaborn as sns\n",
+    "import pandas as pd\n",
+    "from sklearn import svm\n",
+    "from sklearn.ensemble import RandomForestClassifier, RandomForestRegressor, GradientBoostingClassifier\n",
+    "from os import listdir\n",
+    "import random\n",
+    "from os.path import isfile, join\n",
+    "\n",
+    "idx_to_pauli = [\"Z\", \"X\", \"Y\"]\n",
+    "\n",
+    "# Measurement-error-mitigated expectation value prediction\n",
```

Comment:
This should probably be a docstring.  Also, the following code is pretty opaque and could likely use a bit more explanation.